DuckDuckGo/TextZoomCoordinator.swift

```diff
@@ -110,8 +110,7 @@ final class TextZoomCoordinator: TextZoomCoordinating {
     private func applyTextZoom(_ webView: WKWebView) {
         guard isEnabled else { return }
         let level = textZoomLevel(forHost: webView.url?.host)
-        let dynamicTypeScalePercentage = UIFontMetrics.default.scaledValue(for: 1.0)
-        let viewScale = CGFloat(level.rawValue) / 100 * dynamicTypeScalePercentage
```

Comment:
Looking at [the original UserScript implementation](https://github.com/duckduckgo/iOS/pull/3509/files#diff-56a1cce40e39576fa9d081cf40ae93e6f09306ca6340158d448102be50709f50L78-L86), we respect dynamic type for certain known sites (only wikipedia I think). There's probably a historical reason for this.